ps/jenkins/package-testing-ps-build-8.0.yml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: package-testing-ps80-build
+    project-type: pipeline
+    description: |
+        Use package-testing-ps to run in all nodes
+        Do not edit this job through the web!
+    concurrent: true
+    properties:
+        - build-discarder:
+            artifact-days-to-keep: -1
+            artifact-num-to-keep: 10
+            days-to-keep: -1
+            num-to-keep: 50
+    pipeline-scm:
+        scm:
+            - git:
+                url: https://github.com/Percona-Lab/jenkins-pipelines.git
+                branches:
+                    - 'master'
+                wipe-workspace: true
+        lightweight-checkout: true
+        script-path: ps/jenkins/package-testing-ps-build-8.0.groovy
```

Comment:
Please add newline here

Reply:
Added trailing newlines to all the files; thanks for pointing it out!